collectors/ebpf.plugin/ebpf_cachestat.c

```diff
@@ -501,10 +520,11 @@ static void calculate_stats(netdata_publish_cachestat_t *publish) {
  * Sum all values read from kernel and store in the first address.
  *
  * @param out the vector with read values.
+ * @param maps_per_core do I need to read all cores?
  */
-static void cachestat_apps_accumulator(netdata_cachestat_pid_t *out)
+static void cachestat_apps_accumulator(netdata_cachestat_pid_t *out, int maps_per_core)
 {
-    int i, end = (running_on_kernel >= NETDATA_KERNEL_V4_15) ? ebpf_nprocs : 1;
```

Comment:
Why are we not checking for `(running_on_kernel >= NETDATA_KERNEL_V4_15)` anymore (here and at other places too)?

Reply:
I removed it, because we are now controlling these events with new option.
While I was developing, I observed that latest kernel `4.14` also have the [option](https://elixir.bootlin.com/linux/v4.14.313/source/include/uapi/linux/bpf.h#L105), and considering that `CentOS 7.9` was porting recent features from newest kernels, this is not more a concern. If during the tests I am running right now I identify a specific kernel version that cannot run with multiple CPUs, I will add a condition when we set the option before to run. :handshake:

Reply:
OK!

Reply:
I can confirm that changes did not affect `CentOS 7` that was my major concern.